client/src/components/Footer.jsx

```diff
@@ -0,0 +1,50 @@
+import { NavLink } from 'react-router-dom';
+import { ReactComponent as Search } from '../icons/search.svg';
+import { ReactComponent as Add } from '../icons/add.svg';
+import styled from 'styled-components';
+
+const Footer = ({}) => {
```

Comment:
Instead of naming it `Footer` I would name it `Navigation` as this is your `Navigation` or am I wrong? A Footer can be more than a navigation. Therefore, I would choose another name.